### PR TITLE
typo(deleting-commands): Fix typos and weird wording

### PR DESCRIPTION
**Pull request by DevSpen**

### CODE REVIEW

File: guide/creating-your-bot/deleting-commands.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ You may have decided that you don't need a command anymore and don't want your u
 
 ## Deleting specific commands
 
-To delete a specific command, you will require its id. Head to **Server Settings -> Integrations -> Bots and Apps** and choose your bot. Then, right click a command and click **Copy ID**.
+To delete a specific command, you will need its id. Head to **Server Settings -> Integrations -> Bots and Apps** and choose your bot. Then, right click a command and click **Copy ID**.
```
Comment (lolollllo) [inline guide/creating-your-bot/deleting-commands.md:11]: could you replace `id` with `ID` looks better that way imo

File: guide/creating-your-bot/deleting-commands.md [hunk 2]
```diff
@@ -66,4 +66,4 @@ rest.put(Routes.applicationCommands(clientId), { body: [] })
 	.catch(console.error);
 ```
 
-The Discord bot API does currently not provide an easy way to delete guild-based commands that occurr on multiple guilds from all places at once. Each will need a call of the above endpoint, while specifying the respetive guild and command id. Note, that the same command will have a different id, if deployed to a different guild!
+Discord's API doesn't currently provide an easy way to delete guild-based commands that occur on multiple guilds from all places at once. Each will need a call of the above endpoint, while specifying the respective guild and command id. Note, that the same command will have a different id, if deployed to a different guild!
```
Comment (lolollllo) [inline guide/creating-your-bot/deleting-commands.md:69]: could you replace `id` with `ID` looks better that way imo

**Review by lolollllo — CHANGES_REQUESTED** [review]

File: guide/creating-your-bot/deleting-commands.md (hunk 2 above)
Comment (iCrawl) [reply guide/creating-your-bot/deleting-commands.md:69]: No

File: guide/creating-your-bot/deleting-commands.md (hunk 1 above)
Comment (iCrawl) [reply guide/creating-your-bot/deleting-commands.md:11]: No.

File: guide/creating-your-bot/deleting-commands.md (hunk 2 above)
Comment (DevSpen) [reply guide/creating-your-bot/deleting-commands.md:69]: What’s wrong?

Comment (Jiralite) [reply guide/creating-your-bot/deleting-commands.md:69]: Worry not, the original review was deleted.

Comment (DevSpen) [reply guide/creating-your-bot/deleting-commands.md:69]: Oh 😂

**Pull request merged**